src/index.html

```diff
@@ -10,13 +11,105 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
     <title>Moyo header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
+  <!-- #endregion -->
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#">
```

Comment:
The `href` attribute in the `<a>` tag should not contain spaces. Consider using a more descriptive link or an anchor link starting with `#`.